docs/src/content/docs/ja/components/code.mdx

```diff
@@ -76,7 +76,7 @@ export const highlights = ['ファイル', 'CMS'];
 
 ### インポートしたコードを表示する
 
-MDXファイルやAstroコンポーネント内で、[Viteの`?raw`インポートサフィックス](https://vitejs.dev/guide/assets#importing-asset-as-string)を使用して、任意のコードファイルを文字列としてインポートできます。
+MDXファイルやAstroコンポーネント内で、[Viteの`?raw`インポートサフィックス](https://vite.dev/guide/assets#importing-asset-as-string)を使用して、任意のコードファイルを文字列としてインポートできます。
```

Comment:
While here, I guess some of these could point to the translated docs (the Korean and Chinese translations already do):

```suggestion
MDXファイルやAstroコンポーネント内で、[Viteの`?raw`インポートサフィックス](https://ja.vite.dev/guide/assets#importing-asset-as-string)を使用して、任意のコードファイルを文字列としてインポートできます。
```